### ISSUE TITLE
Implement full support for random filters

### ISSUE THREAD

**Issue by richardolsson** [zetkin/organize.zetk.in#404]
The `random` query filter has support for seeding and fractional sizes, which are not yet supported in the Organize UI.

**Comment by richardolsson** [comment]
From the query DSL documentation:

### Random (`random`)
The random filter selects a random subset of the people in an organization. The
size of the subset is defined in the config.

```json
{
    "type": "random",
    "config": {
        "size": 0.7,
        "seed": "abc123"
    }
}
```

The `size` property defines how many people to return and must be a positive
number. Integers above 1 are interpreted as a fixed size, meaning that a value
of 5 will result in at most 5 persons being returned. Fractions between 0 and 1
are interpreted as a percentage of the total number of people in the
organization, so a value of 0.5 will return 50% of the people.

The `seed` property can be used to refresh the random selection. The selection
is stable in the sense that given the same seed, the same subset will always be
returned, as long as the database doesn't change.

If the seed changes, the subset will be very different. If the database changes,
e.g. by adding or removing people, the random subset will at most change by the
number of people added and removed.

The default seed is the empty string.

**Comment by richardolsson** [comment]
Suggested improvements to the current UI:

- [ ] Add a button to refresh the random selection (update the seed)
- [ ] Clarify choice between percentage/exact number (e.g. using a `<select>` to choose method)

**Comment by richardolsson** [comment]
Fixed by #927
